src/js/lib/client/entitlements.mac.plist

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you add detailed docs here - what is this file, what is it for + links to official docs about dev of such files, please?